Source/WarpX.cpp

```diff
@@ -1002,12 +1006,17 @@ WarpX::AllocLevelMFs (int lev, const BoxArray& ba, const DistributionMapping& dm
         Efield_aux[lev][0].reset( new MultiFab(nba,dm,ncomps,ngE));
         Efield_aux[lev][1].reset( new MultiFab(nba,dm,ncomps,ngE));
         Efield_aux[lev][2].reset( new MultiFab(nba,dm,ncomps,ngE));
+        
```

Comment:
Here the M-multifabs are generated for nodal box-arrays. Do we need them there. May be add a comment to mention that currently M-multifabs are not nodal and for most use-cases, we wont need them to be nodal

Reply:
I removed these 3 lines in abca1819c760a813e34dace178bbf70d784671f3